LOAN/LoanContracts/LoanHMDA.rdf

```diff
@@ -198,6 +194,14 @@ Copyright (c) 2015-2018 Object Management Group, Inc.</sm:copyright>
 		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HMDA_Disposition"/>
 	</owl:NamedIndividual>
 	
+	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;_HowSubmitted_directly">
+		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HowSubmitted"/>
```

Comment:
Needs label, definition etc.
What does "directly" even mean?

Reply:
Just ignore this one for now - I didn't review the content and need an SME to fix it.